Makefile

```
@@ -138,12 +138,29 @@ endif
 test: $(SETUP_ENVTEST) ## Run tests
 	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test -v ./... $(TEST_ARGS)
 
+E2E_TEMPLATES_DIR=test/e2e/data/infrastructure-openstack
+E2E_KUSTOMIZE_DIR=test/e2e/data/kustomize
+
+.PHONY: e2e-templates
+e2e-templates: $(addprefix $(E2E_TEMPLATES_DIR)/, \
```

```suggestion
e2e-templates: $(addprefix $(E2E_TEMPLATES_DIR)/, \ ## Generate e2e templates
```

To have a proper usage info